### PR TITLE
fix: encryption status title

### PR DESCRIPTION
**Pull request by gorjan5sk**

### CODE REVIEW

**Review by mobitar — CHANGES_REQUESTED** [review]
Needs subtitle when no encryption available.

**Comment by gorjan5sk** [comment]
Ah, there is a status, but there's also a bug. I'm reusing the status from the account menu, but it's not set unless the account menu has been opened. I'm on it @mobitar .

**Comment by gorjan5sk** [comment]
Ok, now it should be reliable.

File: app/assets/javascripts/preferences/panes/Encryption.tsx [hunk 1]
```diff
@@ -35,14 +36,25 @@ const EncryptionEnabled: FunctionComponent<{ appState: AppState }> = observer(({
   );
 });
 
-export const EndToEndEncryption: FunctionComponent<{ appState: AppState }> = observer(({ appState }) => {
+export const Encryption: FunctionComponent<{ appState: AppState }> = observer(({ appState }) => {
+  const app = appState.application;
+  const hasUser = app.hasAccount();
+  const hasPasscode = app.hasPasscode();
+  const isEncryptionEnabled = hasUser || hasPasscode;
```
Comment (mobitar) [inline app/assets/javascripts/preferences/panes/Encryption.tsx:43]:
```suggestion
  const isEncryptionEnabled = app.isEncryptionAvailable();
```

**Review by mobitar — APPROVED** [review]

**Pull request merged**